styles/index.css

```diff
@@ -1,3 +1,62 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-text {
+  position: absolute;
+  color: rgb(234, 234, 233);
+  top: 35%;
+  left: 5%;
+  font-family: var(--paragraph-sans);
+  font-weight: lighter;
+  text-align: left;
+}
+.overlay-text p {
+  font-size: 140px;
+  margin-bottom: 50px;
+}
+
+.lets-go-button {
+  cursor: pointer;
+  border: 1px solid var(--overlaid-text-color);
+  background-color: transparent;
+  color: var(--overlaid-text-color);
+  font-family: var(--heading-sans);
+  font-size: 14px;
+  letter-spacing: 1px;
+  display: inline-flex;
```

Comment:
what does inline-flex do?

Reply:
Inline-flex prevents the unwanted styles, here underline as the button is inside an anchor tag.